package.json

```diff
@@ -386,4 +386,4 @@
 		],
 		"welcome-build-command": "npm run dev"
 	}
-}
+}
```

Comment:
This change shouldn't affect anything, but I think it's a good idea to revert unnecessary changes.